### PR TITLE
[Epson] Update and reactivate ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by J0WI** [comment]
On epson.com you can find all ccTLD.

**Comment by J0WI** [comment]
@Bisaloo would you like to add those ccTLDs?

**Comment by Bisaloo** [comment]
@J0WI, I'm not sure where to find that list.

**Comment by J0WI** [comment]
> @J0WI, I'm not sure where to find that list.

https://www.epson.com/?_re=1

**Comment by Bisaloo** [comment]
I don't have time to add the rest right now.

File: src/chrome/content/rules/Epson.xml [hunk 1]
```diff
@@ -1,85 +1,146 @@
...
+
+	<target host="epson.se" />
+	<target host="www.epson.se" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://www\.epson\.(cl|co\.cr|com\.(ar|bo|br|cn|co|do|ec|jm|mx|pe|py|uy|ve))/"
+		to="https://epson.$1/" />
+
+	<rule from="^http://epson\.(be|ch|cz|de|dk|fr|ie|it|nl|no|pl|se|com\.tr|co\.uk)/"
```
Comment (J0WI) [inline src/chrome/content/rules/Epson.xml]: Please add a comment for the issues.

**Comment by Bisaloo** [comment]
> Travis exposed some errors.

I'm not sure what's wrong.

**Comment by J0WI** [comment]
`www.epson.com.co` redirects to the invalid host `prod-balancer[12].ea.epson.net`, but as far as I can see that should be fixed by this ruleset.

**Comment by zoracon** [comment]
Closing due to no revisions made. Can reopen if interest for revision is indicated

**Pull request closed without merging**